src/index.js

```diff
@@ -35,6 +36,10 @@ export default class SaucelabsConnector {
             retries:    WEB_DRIVER_CONFIGURATION_RETRIES,
             timeout:    WEB_DRIVER_CONFIGURATION_TIMEOUT
         });
+
+        this.options = {
+            showBrowserStartMessage: options.showBrowserStartMessage

```

Comment:
looks good, but I think we will get the final decision when we have the full list of new options. I suppose you will add them in this pr, right?